woocommerce/single-product/add-to-cart/grouped.php

```diff
@@ -50,7 +51,7 @@
 					$show_add_to_cart_button = true;
 				}
 
-				echo '<tr id="product-' . esc_attr( $grouped_product_child->get_id() ) . '" class="woocommerce-grouped-product-list-item ' . esc_attr( implode( ' ', wc_get_product_class( '', $grouped_product_child ) ) ) . '">';
+				echo '<tr id="product-' . esc_attr( $grouqped_product_child->get_id() ) . '" class="woocommerce-grouped-product-list-item ' . esc_attr( implode( ' ', wc_get_product_class( '', $grouped_product_child ) ) ) . '">';
```

Comment:
typo in the variable name `grouped` vs `grouqped`

Reply:
thanks